src/mame/drivers/model2.cpp

```diff
@@ -1684,6 +1684,16 @@ static INPUT_PORTS_START( model2 )
 	PORT_BIT(0x20, IP_ACTIVE_LOW, IPT_JOYSTICK_UP)    PORT_PLAYER(2)
 	PORT_BIT(0x40, IP_ACTIVE_LOW, IPT_JOYSTICK_RIGHT) PORT_PLAYER(2)
 	PORT_BIT(0x80, IP_ACTIVE_LOW, IPT_JOYSTICK_LEFT)  PORT_PLAYER(2)
+
+	PORT_START("DSW")
+	PORT_DIPUNUSED_DIPLOC(0x01, 0x01, "DSW:1")
+	PORT_DIPUNUSED_DIPLOC(0x02, 0x02, "DSW:2")
+	PORT_DIPUNUSED_DIPLOC(0x04, 0x04, "DSW:3")
+	PORT_DIPUNUSED_DIPLOC(0x08, 0x08, "DSW:4")
+	PORT_DIPUNUSED_DIPLOC(0x10, 0x10, "DSW:5")
+	PORT_DIPUNUSED_DIPLOC(0x20, 0x20, "DSW:6")
+	PORT_DIPUNUSED_DIPLOC(0x40, 0x40, "DSW:7")
+	PORT_DIPUNUSED_DIPLOC(0x80, 0x80, "DSW:8")
```

Comment:
Wouldn't it be better to default them to “unknown” rather than “unused” since at least some of them are used for for debug features in some games?  If they’re found to be unused in certain games, they can modify the port to mark the actual unused switches as such.

What is the actual silkscreen label for the DIP switch pack?  In `zerogun` it’s currently labeled “SW” but you‘ve used “DSW” here.  It should match the silkscreen, manual and/or schematics.

Reply:
> Wouldn't it be better to default them to “unknown” rather than “unused” since at least some of them are used for for debug features in some games? If they’re found to be unused in certain games, they can modify the port to mark the actual unused switches as such.
That's a good point, I'll update that.

> What is the actual silkscreen label for the DIP switch pack? In `zerogun` it’s currently labeled “SW” but you‘ve used “DSW” here. It should match the silkscreen, manual and/or schematics.
I wasn't aware of the naming rules around that. I'll follow up and make sure (it's likely SW like Zero Gunner).